changelog.d/+pdm.changed.md

```diff
@@ -0,0 +1 @@
+Package the library using [pdm](https://pdm-project.org).
```

Comment:
is this going to affect the user? if we still expect that `pip install` will work exactly the same for them I would change category to `.infrastructure.`

Reply:
I understand `infrastructure` category as things that do not affect the end product at all, like CI. Here, hopefully the change won't matter to end users but it does affect the source distribution at least to some extent.